nkms/crypto/constants.py

```diff
@@ -1,5 +1,15 @@
-# TODO: Turn these into classes?
-HASH_DIGEST_LENGTH = 32
+from cryptography.hazmat.primitives import hashes
+BLAKE2B = hashes.BLAKE2b(64)
```

Comment:
I don't think this should be in this file, but it can stay for now. Let's think of a better place for it.